app/models/edition/first_image_pulled_out.rb

```diff
@@ -13,4 +13,8 @@ module Edition::FirstImagePulledOut
   def image_disallowed_in_body_text?(index)
     index == 1
   end
+
+  def first_image_pulled_out?
+    true
```

Comment:
I think maybe using `@edition.include?(Edition::FirstImagePulledOut)` in the component class might be more extensible than having to define this method on all document types that will need to go through this in the future?